src/components/SubscribeNotification/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import BellWhiteImage from 'common/icons/bellWhite.svg';
+import BellBlackImage from 'common/icons/bellBlack.svg';
+import PropTypes from 'prop-types';
+import styles from './NotificationBell.module.css';
+import cn from 'classnames';
+
+const SubscribeNotification = ({ hasSubscribed = false } = {}) => {
+  return (
+    <div
```

Comment:
Optional: 建議用 button 以達到更清楚的語意
```suggestion
    <button
```

Reply:
對，你以前也提醒過幾次了，我改一下